Library/Homebrew/tap.rb

```diff
@@ -229,7 +229,7 @@ def install(options = {})
       raise
     end
 
-    link_manpages
```

Comment:
This allows us to add a unit test for the result of `brew --repair`. See https://github.com/Homebrew/brew/pull/1195/files/d0539cd93c826f0129b2dae148761424d59a94e9#diff-3676702e08834bbec8ee89f8f9869981R279

Would you be more likely to accept this if it were named just `repair`?